travel_temp.html

```diff
@@ -31,6 +31,9 @@
         <meta name="summary"
             content="An open source project aimed at providing free and beautiful templates to everyone for building their portfolio websites and showcase their work to the world." />
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" />
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Noto+Serif:ital@1&display=swap" rel="stylesheet">
 
         <!--For Dark/Light mode Toggle-->
         <script src="https://code.iconify.design/1/1.0.4/iconify.min.js"></script>
```

Comment:
This is a straightforward task. You were supposed to just replace the font family. (Check Line 41 below. Not satisfied)